tests/phpunit/api/v4/Api4TestBase.php

```diff
@@ -53,6 +53,7 @@ public function setUpHeadless(): CiviEnvBuilder {
    */
   public function tearDown(): void {
     $this->conditionallyDeleteTestRecords();
+    \CRM_Utils_Time::resetTime();
```

Comment:
This won't give the error, but probably should be calling parent.

Reply:
Good catch, fixed.